model/model.go

```diff
@@ -87,6 +88,27 @@ func (c *ColumnInfo) IsGenerated() bool {
 	return len(c.GeneratedExprString) != 0
 }
 
+// SetDefaultValue sets the default value.
+func (c *ColumnInfo) SetDefaultValue(value interface{}) error {
+	c.DefaultValue = value
+	if c.Tp == mysql.TypeBit {
+		if v, ok := value.(string); ok {
+			c.DefaultValueBit = []byte(v)
+			return nil
+		}
+		return types.ErrInvalidDefault.GenByArgs(c.Name)
+	}
+	return nil
+}
+
+// GetDefaultValue gets the default value.
```

Comment:
I feel like we need to add more comments. I understand that `GetDefaultValue` will return the default value of the column but I am more interested in the case handling logic.

Reply:
`GetDefaultValue` just return the column's default value.
just use `column.GetDefaultValue()` instead of `column.DefaultValue`,
because the bit type default value will stored in `DefaultValueBit` field.